vivisect/qt/memory.py

```diff
@@ -141,7 +141,6 @@ def _hotkey_nav_nextundef(self):
         if vw.isValidPointer(va+lastloc[1]):
             va += lastloc[1]
         self._navExpression(hex(va))
-        self._selectVa(va)
```

Comment:
have we figured out a good way to unittest gui stuff?  maybe a StringCanvas?  i doubt CI's allow Qt stuff to go...  but it'd be nice to have a unittest that navigates to a place, changes where the canvas view area is and then query it from python to make sure it's where it should be.